.travis.yml

```diff
@@ -17,5 +25,8 @@ matrix:
         script: Rscript -e 'pkgdown::deploy_site_github(verbose = TRUE)'
         skip_cleanup: true
     - r: oldrel
+      env: GARGLE_PASSWORD=''
     - r: 3.3
+      env: GARGLE_PASSWORD=''
     - r: 3.2
+      env: GARGLE_PASSWORD=''
```

Comment:
In bigrquery, you run auth-requiring tests on 3.2 and release. Is that because you sort of want to "bookend" things? You skip auth-requiring tests on the versions in between.

Reply:
I'm not sure what I was thinking — I suspect I meant to also not run auth on 3.2. Some more cogent thoughts:

Tests that require auth tend to be slower (since they're going to be sending data back and forth), and the API is independent of the R version, so all else being equal it seems that you only need to run on R-release. 

I think I also deliberately run on R-release without auth because that's the closest to what CRAN will see, and it seems nice to double check.